src/richText.js

```diff
@@ -394,7 +394,7 @@ define([
 
         // only support absolute path right now
         if (!form.getMugByPath(xpath) && !/instance\('casedb'\)/.test(xpath)) {
-            return value.replace('<', '&lt;').replace('>', '&gt;');
+            return $('<span>').text(value);
```

Comment:
Thanks for the explanations. That makes a lot of sense that it `replacePathWithBubble()` needs to return a jquery object but it wasn't in some cases.

However, I think the resulting non-bubble thing should not be wrapped in a span tag. It's a little tricky to accomplish this. PR coming...